### PR TITLE
Fix eslint errors

### PR DESCRIPTION
**Pull request by deBhal**
This PR is actually a place for me to experiment with statuses and github webooks hooks for #hackday, but I figured I may as well fix some eslint errors while I'm at it.

### CODE REVIEW

**Comment by deBhal** [comment]
I'll flag the changes as Needs Review.  I'm sure I can find something else trivial to fix for more experiments.

File: server/devdocs/index.js [hunk 1]
```diff
@@ -59,18 +59,18 @@ function listDocs( filePaths ) {
...
+		return {
+			path: path,
+			title: 'Not found: ' + path,
+			snippet: ''
+		};
 	} );
 	return results;
 }
 
-/**
```
Comment (retrofox) [inline server/devdocs/index.js]: Why are we removing the last `*`? We have a lot documentation blocks there similar to this one. Also -> http://usejsdoc.org/about-block-inline-tags.html

Comment (retrofox) [inline server/devdocs/index.js]: Oh, I see. it ins't a function block. My bad.

**Comment by retrofox** [comment]
Ignore me. It just isn't a function comment block. LGTM :ship:

File: server/devdocs/index.js (hunk 1 above)
Comment (deBhal) [inline server/devdocs/index.js]: Yeah, it is a function but it's not exported so I went with "make it just a comment" here.

**Pull request merged**